akash.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Akash</title>
+</head>
+<body>
+    <form>
+        <div class="Login">
+            <div class="box">                
+                <p>login</p>
+                <input type="text" placeholder="User name" required="">
+                <input type="Password" placeholder="Password" required="">
+                <div class="forget">forget password</div>
+                <button type="submit" class="submit">Login</button>
```

Comment:
why you add submit button?

Reply:
Because it is a button, so I change into the submit button